.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,58 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+
+# Same name as the file for consistent identification
+name: pre-commit
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "*"
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  pre-commit:
+    name: Detecting code style issues
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install ALSA and Jack dependencies
+        run: |
+          sudo apt-get update && sudo apt-get install -y libasound2-dev libjack-jackd2-dev
```

Comment:
Jack is not strictly necessary to build the lib (but it won't hurt either). We don't install it in the `rust.yaml` run

Reply:
This way we could simply use `--all-features` for the checks.